.github/workflows/ansible-test.yml

```diff
@@ -8,6 +8,9 @@ name: CI
 on:
   # Run CI against all pushes (direct commits, also merged PRs), Pull Requests
   push:
+    branches:
+      - main
+      - stable-*
```

Comment:
Without this, every PR created by the patchback bot will receive both push and pull_request CI jobs, which is a waste of CI resources.

Reply:
@felixfontein This wasn't present originally because with such a change you steal the ability to test changes from the contributors, in their forks. Instead, you could use exclusions maybe.